### PR TITLE
Usability & accuracy commits

### PR DESCRIPTION
**Pull request by JeremieGomez**
See each commit for the changed content, I tried to keep them small.

### CODE REVIEW

File: bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java [hunk 1]
```diff
@@ -94,9 +112,15 @@ public static void main(String[] args) throws ParseException, IOException {
     logger.info(
         "Processing finished."
```
Comment (franklinWhaite) [inline bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java:113]: ``
```suggestion
        "Processing finished. "
```

File: bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java [hunk 2]
```diff
@@ -94,9 +112,15 @@ public static void main(String[] args) throws ParseException, IOException {
...
             + "Queries read: {}. "
+            + "Fully parsed queries: {}. "
+            + "Partially parsed queries: {}. "
```
Comment (franklinWhaite) [inline bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java:116]: What is the difference between partially parsed and fully parsed?

File: bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java [hunk 3]
```diff
@@ -94,9 +112,15 @@ public static void main(String[] args) throws ParseException, IOException {
...
+            + "Queries that could not be parsed: {}. "
             + "Queries with anti-patterns: {}. "
-            + "Queries that could not be parsed: {}.",
+            + "Queries with another error: {}.",
```
Comment (franklinWhaite) [inline bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java:119]: What is the difference between "that could not be parsed" and "another error"

File: bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/cmd/BQAntiPatternCMDParser.java [hunk 4]
```diff
@@ -220,20 +231,11 @@ public Iterator<InputQuery> getInputQueries() {
...
-          cmd.getOptionValue(READ_FROM_INFO_SCHEMA_TABLE_OPTION_NAME));
-    } else if (cmd.hasOption(READ_FROM_INFO_SCHEMA_FLAG_NAME)) {
-      return new InformationSchemaQueryIterable(
-          cmd.getOptionValue(PROCESSING_PROJECT_ID_OPTION_NAME),
-          cmd.getOptionValue(READ_FROM_INFO_SCHEMA_DAYS_OPTION_NAME));
-    } else {
-      return new InformationSchemaQueryIterable(
-          cmd.getOptionValue(PROCESSING_PROJECT_ID_OPTION_NAME));
-    }
+    String processingProjectId = cmd.getOptionValue(PROCESSING_PROJECT_ID_OPTION_NAME);
```
Comment (franklinWhaite) [inline bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/cmd/BQAntiPatternCMDParser.java:234]: nice refactor!

File: README.md [hunk 5]
```diff
@@ -220,14 +221,22 @@ To read input queries from INFORMATION_SCHEMA.JOBS.
...
 Defaults to 1.
 </ul>
 
-``--read_from_info_schema_days="\`region-us\`.INFORMATION_SCHEMA.JOBS"``
+``--info_schema_table_name="\`region-us\`.INFORMATION_SCHEMA.JOBS"``
 <ul>
 Specifies what variant of INFORMATION_SCHEMA.JONS to read from.
 </ul>
 
+``--info_schema_min_slotms=n``
```
Comment (franklinWhaite) [inline README.md:233]:
```suggestion
``--info_schema_min_slot_ms=n``
```

File: README.md [hunk 6]
```diff
@@ -220,14 +221,22 @@ To read input queries from INFORMATION_SCHEMA.JOBS.
...
+<ul>
+Specifies a minimum number of slotms (integer value).<br> The queries that consume less than this minimum will not be selected for processing.
```
Comment (franklinWhaite) [inline README.md:235]:
```suggestion
Specifies a minimum number of slot_ms (integer value).<br> The queries that consume less than this minimum will not be selected for processing.
```

File: bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java (hunk 2 above)
Comment (JeremieGomez) [reply bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java:116]: This is the new feature, I think the most important one.
If all parsers were able to parse the query with no issue, the query is fully parsed. If only some parsers (at least one) is able to parse the query with no issue, the query is partially parsed. If 0 parser was able to parse the query with no issue, it is not parsed.

File: bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java (hunk 3 above)
Comment (JeremieGomez) [reply bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java:119]: Could not be parsed is if 0 parsers were able to parse the query.
Other error is if no issue is found with the parsers, but the analysis falls in the global catch block. Then we can know that some queries had problems.

**Review by franklinWhaite — APPROVED** [review]
LGTM

**Pull request merged**